### ISSUE TITLE
Feature request: Add an option to disable the address bar icons

### ISSUE THREAD

**Issue by Aleksandra0056** [filips123/PWAsForFirefox#508]
### Problem Description

I like the address bar icon (Install this site/Launch this site), but I would like to have the option to disable it since I can install & launch PWAs directly from the PWAsForFirefox menu (or launch from the start menu, desktop & taskbar). Also, the "display icon with arrow" (Install this site) is with square corners style and isn't following Proton design, but that's a thing for another issue.

But my problem isn't entirely with the style, but with address bar clutter. This isn't a problem with PWA instances but with Firefox itself.

### Proposed Solution

Add a single option to disable address bar icons (Install this site/Launch this site) to PWAsForFirefox settings (gear icon menu).

### Additional Information

It has been a very long time since I regularly reported issues on GitHub, so you can expect incomplete information or duplicates (I apologise for any issues and I will try to avoid them as much as possible).

EDIT: I'm attaching screenshots of the icons I mentioned:

![image](https://github.com/filips123/PWAsForFirefox/assets/69351811/0f76ca03-54f4-4715-a958-d28a9a89f371)

![image](https://github.com/filips123/PWAsForFirefox/assets/69351811/86819e72-d390-4d3e-8911-fc968f5e529f)

**Comment by filips123** [comment]
You can already set "Display address bar widget" to never in the extension settings to hide these icons.

**Comment by Aleksandra0056** [comment]
> You can already set "Display address bar widget" to never in the extension settings to hide these icons.

I'm apologising. I didn't notice the option on PWAs options, because it's labelled as "Display address bar widget" instead of "Display address bar icon", but since I started using MS Edge a few months ago, and I switched to Firefox one week ago, then I absolutely forgot about many functions in my extensions and apps.

Today I also reported one invalid issue in Bugzilla about a feature which already worked as intended. I don't know what happened to me, because I almost always reported valid issues & feature requests in past.

I'm apologising again and I wish you a good day. I'm closing this as not planned.
